main/views.py

```diff
@@ -1,6 +1,14 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.contrib.auth import logout
 
 
 # Create your views here.
 def homePage(request):
+    if request.user.is_authenticated:
+        return render(request, 'main/loggedInPage.html')
```

Comment:
Why do you need a new template for this? Maybe this can be part of the base, something like
```
{% if user.is_authenticated %}
{% extends 'loggedInPage.html' %}
{% endif %}
```

Reply:
Im not sure if I understand the template reference correctly, the second line in the code snippet you wrote uses the loggedInPage.html, doesn't that mean that the template (file) is needed?

I decided to create a separate template to keep it more modular, similar to the homePage template (which extends the base_home when there is no logged in user). Additionally, we consider changing the design of the base_home in the future (the top navbar and background), so keeping it minimal will help.

What do you think?

Reply:
Do you want the side panel to show on every page you have? If so I think it should be in the base (can extend this)
If it's just for the homepage, maybe you can extend this template in the `homePage` so you will have only one template rendered

Reply:
As discussed offline, the loggedInPage is a different page than the homePage (default screen), and since we can't extend as suggested above we leave it as two files.